### PR TITLE
Adding ocpShortVersion and benchmark details

### PR DESCRIPTION
**Pull request by vishnuchalla**
## Type of change

- [x] Refactor
- [ ] New feature
- [x] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Adding few more details like `ocpShortVersion` and `benchmark` details in CI data. Removing nodeName as it is confusing.

OCP short version logic was decided based on the release patterns here: https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Review by mukrishn — APPROVED** [review]
lgtm

**Comment by vishnuchalla** [comment]
@jtaleric just a gentle reminder. PTAL 👋🏼

File: workloads/ingress-perf/run.sh [hunk 1]
```diff
@@ -40,5 +40,5 @@ if [ $exit_code -eq 0 ]; then
 else
   JOB_STATUS="failure"
 fi
-env JOB_START="$JOB_START" JOB_END="$JOB_END" JOB_STATUS="$JOB_STATUS" UUID="$UUID" ES_SERVER="$ES_SERVER" ../../utils/index.sh
+env JOB_START="$JOB_START" JOB_END="$JOB_END" JOB_STATUS="$JOB_STATUS" UUID="$UUID" WORKLOAD="$WORKLOAD" ES_SERVER="$ES_SERVER" ../../utils/index.sh
```
Comment (jtaleric) [inline workloads/ingress-perf/run.sh:43]: Where is $WORKLOAD defined?

File: workloads/kube-burner-ocp-wrapper/run.sh [hunk 2]
```diff
@@ -122,5 +122,5 @@ if [ $exit_code -eq 0 ]; then
 else
   JOB_STATUS="failure"
 fi
-env JOB_START="$JOB_START" JOB_END="$JOB_END" JOB_STATUS="$JOB_STATUS" UUID="$UUID" ES_SERVER="$ES_SERVER" ../../utils/index.sh
+env JOB_START="$JOB_START" JOB_END="$JOB_END" JOB_STATUS="$JOB_STATUS" UUID="$UUID" WORKLOAD="$WORKLOAD" ES_SERVER="$ES_SERVER" ../../utils/index.sh
```
Comment (jtaleric) [inline workloads/kube-burner-ocp-wrapper/run.sh:125]: Where is $WORKLOAD defined?

File: workloads/network-perf-v2/run.sh [hunk 3]
```diff
@@ -39,5 +39,5 @@ if [ $run -eq 0 ]; then
 else
   JOB_STATUS="failure"
 fi
-env JOB_START="$JOB_START" JOB_END="$JOB_END" JOB_STATUS="$JOB_STATUS" UUID="$UUID" ES_SERVER="$ES_SERVER" ../../utils/index.sh
+env JOB_START="$JOB_START" JOB_END="$JOB_END" JOB_STATUS="$JOB_STATUS" UUID="$UUID" WORKLOAD="$WORKLOAD" ES_SERVER="$ES_SERVER" ../../utils/index.sh
```
Comment (jtaleric) [inline workloads/network-perf-v2/run.sh:42]: hm, WORKLOAD in this case is smoke.yaml or full.yaml which isn;t all to useful, we might just want to prefix `k8s-netperf`

Comment (jtaleric) [reply workloads/network-perf-v2/run.sh:42]: or maybe `netperf`? thoughts?

File: workloads/ingress-perf/run.sh (hunk 1 above)
Comment (vishnuchalla) [reply workloads/ingress-perf/run.sh:44]: added logic to populate it.

File: workloads/network-perf-v2/run.sh (hunk 3 above)
Comment (vishnuchalla) [reply workloads/network-perf-v2/run.sh]: Adding it as `k8s-netperf` according to our tool name.

File: workloads/network-perf-v2/run.sh [hunk 4]
```diff
@@ -39,5 +39,5 @@ if [ $run -eq 0 ]; then
 else
   JOB_STATUS="failure"
 fi
-env JOB_START="$JOB_START" JOB_END="$JOB_END" JOB_STATUS="$JOB_STATUS" UUID="$UUID" ES_SERVER="$ES_SERVER" ../../utils/index.sh
+env JOB_START="$JOB_START" JOB_END="$JOB_END" JOB_STATUS="$JOB_STATUS" UUID="$UUID" WORKLOAD="k8s-netperf-${WORKLOAD%.yaml}" ES_SERVER="$ES_SERVER" ../../utils/index.sh
```
Comment (jtaleric) [inline workloads/network-perf-v2/run.sh:42]: lets just go with `k8s-netperf`

Comment (vishnuchalla) [reply workloads/network-perf-v2/run.sh]: Acknowledged. Changed as suggested.

**Review by jtaleric — APPROVED** [review]
lgtm

**Pull request merged**